doc/faq.rst

```diff
@@ -195,6 +195,41 @@ system level, not the user level. In the example above, this means changing
 See the man page for :code:`ch-run` for more on environment variable
 handling.
 
+:code:`ch-run` fails with “can't execve(2): permission denied”
+--------------------------------------------------------------
+
+For example::
+
+  $ ch-run /var/tmp/hello -- /bin/echo foo
+  ch-run[154334]: error: can't execve(2): /bin/echo: Permission denied (ch_core.c:387 13)
+
+But :code:`/bin/echo` *does* have execute permission::
+
+  $ ls -lh /var/tmp/hello/bin/echo
+  -rwxr-xr-x 1 charlie charlie 51 Oct  8  2021 /var/tmp/hello/bin/echo
+
+In this case, the error indicates the container image is on a filesystem
+mounted with :code:`noexec`. To verify this, you can use e.g.
+:code:`findmnt(8)`::
+
+  $ findmnt
+  TARGET      SOURCE  FSTYPE  OPTIONS
+  [...]
+  └─/var/tmp  tmpfs   tmpfs   rw,noexec,relatime,size=8675309k
+
+Note :code:`noexec` under :code:`OPTIONS`.
+
+To fix this, you can:
+
+  1. Use a different filesystem mounted :code:`exec` (i.e., the opposite
+     of :code:`noexec` and typically the default).
+
+  2. Change the mount options for the filesystem.
```

Comment:
I would reword to to say: "Remount with exec options for the filesystem" or something like that.